app/assets/javascripts/discourse/app/controllers/preferences/second-factor.js

```diff
@@ -154,9 +130,13 @@ export default Controller.extend(CanCheckEmails, {
 
       this.dialog.deleteConfirm({
         title: I18n.t("user.second_factor.disable_confirm"),
-        message: this.disableAllMessage(),
+        bodyComponent: "dialog-messages/second-factor-confirm-phrase",
```

Comment:
I tried passing the name of the component using the angle-bracket format here (i.e. `DialogMessages::SecondFactorConfirmPhrase`), but it didn't work. Not sure if it's a limitation, would be nice for component name syntax to be the same across JS/HBS files.